frontend/src/components/StakeSelectorItem.vue

```diff
@@ -77,8 +92,23 @@ export default {
     },
     rewardsDurationFormatted() {
       return formatDurationFromSeconds(this.rewardsDuration);
-    }
-  }
+    },
+  },
+  methods: {
+    getExclamationMark(rewardDistributionTimeLeft) {
+      if(rewardDistributionTimeLeft > 0) {
+        return 'green-exclamation-mark';
+      }
+      return 'red-exclamation-mark';
+    },
+
+    getTooltip(rewardDistributionTimeLeft) {
+      if(rewardDistributionTimeLeft > 0) {
+        return 'rewards still available here';
```

Comment:
Please use i18n translations. Check readme on how to use it or look into other components that have translations